expression/bench_test.go

```diff
@@ -200,6 +200,52 @@ type dataGenerator interface {
 	gen() interface{}
 }
 
+type defaultGener struct {
+	nullRation float64
+	eType      types.EvalType
+}
+
+func (g *defaultGener) gen() interface{} {
```

Comment:
Do hybrid types needs to be handled here?

Reply:
How about supporting hybrid types later or letting our contributors to support this feature?